src/Util/Str.php

```diff
@@ -1,9 +1,10 @@
 <?php
 
-/*
+/**
  * This file is part of Flarum.
+ * Usage: $slug = Str::slug( 'Cómò ünç 中文百强网' , md5(uniqid(rand(), true)) );
  *
- * (c) Toby Zerner <toby.zerner@gmail.com>
+ * @author: Zeokat 
```

Comment:
This should remain as Toby. You didn't create the file, you contributed to it. Every single file that's part of Flarum is Toby's property. You just helped make it better. 😉 
Also, is not a documentation comment. It's the header that is in every single PHP file on Flarum/core, so it shouldn't have the usage either.
I'm pretty sure the usage would go in the method as
```php
 * @example
 * <EXAMPLE>
```